tests/test_login.py

```diff
@@ -1,19 +1,34 @@
-import unittest
 
 
-from trytond.tests.test_tryton import ModuleTestCase
-from trytond.tests.test_tryton import suite as test_suite
+from trytond.tests.test_tryton import DB_NAME
+from werkzeug.test import Client
+from trytond.wsgi import app
+from werkzeug.wrappers import Response
+from werkzeug.wrappers.json import JSONMixin, _JSONModule
 
 
-class AuthenticationDummyTestCase(ModuleTestCase):
-    'Test Authentication Dummy module'
-    module = 'authentication_dummy'
+class JSONResponse(JSONMixin, Response):
+    json_module = _JSONModule
 
-def test_login_dummy(self):
-    self.assertTrue(False)
 
-def suite():
-    suite = test_suite()
-    suite.addTests(unittest.TestLoader().loadTestsFromTestCase(
-            AuthenticationDummyTestCase))
-    return suite
+class LoginTestCase(object):
```

Comment:
Given there will be a single test in this module, I feel like this TestMixin approach to build the test case may be unnecessary this time